packages/components/src/components/headline/headline.tsx

```diff
@@ -2,20 +2,22 @@ import { Component, Element, h, Host, JSX, Prop } from '@stencil/core';
 import {
   AllowedTypes,
   attachComponentCss,
-  attachSlottedCss,
   getDataThemeDarkAttribute,
   THEMES,
   TEXT_ALIGNS,
   validateProps,
+  warnIfComponentIsDeprecated,
 } from '../../utils';
 import type { PropTypes, TextAlign, TextColor, Theme } from '../../types';
-import type { HeadlineTag, HeadlineVariant } from './headline-utils';
-import { getHeadlineTagName, HEADLINE_TAGS } from './headline-utils';
-import { getComponentCss, getSlottedCss } from './headline-styles';
+import type { HeadingTag } from '../heading/heading-utils';
+import type { HeadlineVariantDeprecated } from './headline-utils';
+import { getHeadlineTagName } from './headline-utils';
+import { HEADING_TAGS } from '../heading/heading-utils';
+import { getComponentCss } from '../heading/heading-styles';
```

Comment:
This is probably the root cause for all rollup and dsr component import related changes.
Could easily be solved similar as for link-social.

Reply:
Yes it is the root cause 🙈 so i guess its better to add a styles file again and wrap in another function

Reply:
Better would be to have a look at `link-social-styles.ts` and see how it it is solved there.
https://github.com/porsche-design-system/porsche-design-system/blob/issue/v3-alpha/packages/components/src/components/link-social/link-social-styles.ts